segmenters/nlp/Sentencizer/tests/test_sentencizer.py

```diff
@@ -56,9 +56,41 @@ def validate(req):
         f.index_lines(['  This ,  text is...  . Amazing !!'], on_done=validate, callback_on_body=True, line_format='csv')
 
 
-def test_sentencier_cn():
+def test_sentencier_en_filter():
+    """
+    Filter should still work for English
+    """
     sentencizer = Sentencizer()
+    text = 'It is a sunny day!!!! When müller comes back, we are going to the zoo. 😁'
+    crafted_chunk_list = sentencizer.segment(text, 0)
```

Comment:
why the extra 0?

Reply:
It was there in the tests before. But I guess it is not needed